src/indigo/less/main-content.less

```diff
@@ -1,37 +1,3 @@
-.kbc-main {
```

Comment:
viem, ze este na tom makas, no ten `kbc-main` by som nevyhadzoval, pouziva sa to :)

Reply:
jop díky